manifests/netinstall.pp

```diff
@@ -113,7 +113,7 @@
     default => $postextract_cwd,
   }
 
-  if $preextract_command {
+  if $preextract_command != '' {
```

Comment:
I'd check here also for false values so:
if $preextract_command and $preextract_command != ''